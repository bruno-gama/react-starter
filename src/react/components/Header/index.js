import React from 'react'
import styled from 'styled-components'
import { Heading, Box } from 'rebass'

const Header = () => (
  <Box px={4} py={5} color="white" bg="teal">
    <Heading is="h1" fontSize={[4, 5, 6]}>
      You are beautiful
    </Heading>
    <Heading is="h2" fontSize={[3, 4, 5]}>
      Have a nice day
    </Heading>
  </Box>
)

export default Header
