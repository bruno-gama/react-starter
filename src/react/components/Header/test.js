import React from 'react'
import { render } from '@testing-library/react'

import Header from './'

describe.only('Header', () => {
  test('renders correctly', () => {
    const { container } = render(<Header />)
    expect(container).toMatchSnapshot()
  })
})
