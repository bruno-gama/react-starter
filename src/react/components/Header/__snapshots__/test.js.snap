// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<div>
  <div
    class="sc-bdVaJa gsdBKM"
  >
    <h1
      class="sc-VigVT fBDaOQ"
    >
      You are beautiful
    </h1>
    <h2
      class="sc-VigVT gNCaSo"
    >
      Have a nice day
    </h2>
  </div>
</div>
`;
